### ISSUE TITLE
Golden Axe 3 crash 

### ISSUE THREAD

**Issue by DarkLegend1987** [TASVideos/BizHawk#2076]
Hi team im TASing a Golden Axe 3 but when im launch the game an go into the title screen the game crash im using bizhawk 2.4.2 im test back in a old copy 1.13 of Win XP im have and the game run normal

**Comment by nattthebear** [comment]
I can't seem to reproduce this on the latest master or a 2.4.1 release.
```
Golden Axe III (J) [!]
SHA1:CD9ECC1DF4E01D69AF9BEBCF45BBD944F1B17F9F
MD5:7D9F963C9C2E22B542516EDD03ADA6F6
```

**Comment by DarkLegend1987** [comment]
Yeah friend im launch on old 1.13 and complete my TASing im not sure what happen but is the same SHA1 of the rom im used

**Comment by DarkLegend1987** [comment]
Ok im test this and in 2.4.2 get fixed they can load in 2.4.1

**Comment by YoshiRulz** [comment]
Broken in 2.4.2 and working in 2.4.1 is not fixed, it's a regression.
